src/compute-pr-actions.ts

```diff
@@ -2,44 +2,47 @@ import * as Comments from "./comments";
 import { PrInfo, ApprovalFlags } from "./pr-info";
 import { TravisResult } from "./util/travis";
 
-export type Actions = typeof DefaultActions;
-export const DefaultActions = {

```

Comment:
`computePrActions` was mutating `DefaultActions.labels` on every run. That’s the bug.